src/explore-education-statistics-frontend/src/modules/table-tool/components/Footnote.tsx

```diff
@@ -18,9 +18,11 @@ const Footnote = ({ content }: FootnoteProps) => {
   return (
     <ol className="govuk-list">
       {footnoteList.slice(0, 3)}
-      <Details summary="View additional footnotes">
-        <ol className="govuk-list">{footnoteList.splice(3)}</ol>
-      </Details>
+      {footnoteList.length > 3 && (
+        <Details summary="View additional footnotes">
+          <ol className="govuk-list">{footnoteList.splice(3)}</ol>
```

Comment:
Should use `slice` here instead of `splice`.

`splice` mutates the underlying array and is potentially a cause of bugs. Always prefer to use `slice` unless you have created a copy of the array directly before using `splice`.

Reply:
Changed